lib/tests/formulae.rb

```diff
@@ -510,6 +510,11 @@ def formula!(formula_name, args:)
 
         livecheck(formula) if !args.skip_livecheck? && !skip_online_checks
 
+        if ENV["GITHUB_ACTIONS_HOMEBREW_SELF_HOSTED"].present? && OS.mac? && MacOS.version == :sequoia
+          # Fix intermittent broken disk cache on Sonoma after building from source.
```

Comment:
I've mistaken Sequoia for Sonoma at least 3 times in the past day and it seems I'm not the only one!

Reply:
ugh 🤦

Reply:
#1212